deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -480,7 +480,14 @@ async def get_interactive_waveform_data(
             download_directory_from_s3(bucket=PULSE3D_UPLOADS_BUCKET, key=key, file_path=tmpdir)
 
             # read the time force dataframe from the parquet file
-            parquet_path = glob(os.path.join(tmpdir, "time_force_data", "*.parquet"), recursive=True)
+            # older versions of pulse3d to not inlcude the version data
+            if hasattr(parsed_meta, "version"):
```

Comment:
I think this should be a `dict` in which you can check if a key is present by just doing `if "version" in parsed_meta`. It would be best though in this case to just use `parsed.get("version")` combined with the walrus operator `if pulse3d_version := parsed_meta.get("version"):`.